lib/screens/images_to_pdf.dart

```diff
@@ -74,10 +90,16 @@ class _ImageToPdfState extends State<ImageToPdf> {
             const SliverGridDelegateWithFixedCrossAxisCount(crossAxisCount: 3),
         itemCount: _images.length,
         itemBuilder: (context, index) {
-          return Card(
-            child: Image.file(
-              _images[index],
-              fit: BoxFit.cover,
+          return GestureDetector(
+            onLongPress: () {
+              _images.removeAt(index);
```

Comment:
better add a cross/clear ```IconButton``` and pressing on that button removes the image and ```remove``` is better method because index slides every time